torch/ao/quantization/fx/_decomposed.py

```diff
@@ -0,0 +1,43 @@
+import torch
+from torch.library import Library, impl
+
+quantized_decomposed_lib = Library("quantized_decomposed", "DEF")
```

Comment:
Are you sure you want to name this "quantized_decomposed"? Doesn't seem very descriptive

Reply:
This name is giving me jitters :)